kuma/users/apps.py

```diff
@@ -1,11 +1,9 @@
 import stripe
 from django.conf import settings
 from django.contrib.auth.apps import AuthConfig
+from django.core.checks import register
 from django.utils.translation import gettext_lazy as _
 
-# the checks self-register so we don't need to use anything from the import
-import kuma.users.checks  # noqa: F401
-
```

Comment:
I know we had a debate about this when you originally implemented it. So don't think I'm using this chance to rewrite it in a style/fashion that *I* prefer. 
No, the reason it has to be like this is that if not done like this, the models aren't loaded by the way this *import* happens. the `register()` wrapper would make sure the check isn't actually executed until after the `UserConfig.ready()` is executed, but it's different about import-time stuff. I think you can work through the exactly order and logic. If you simply put in a temporary `import kuma.users.checks` here and start Django you'll see the error for yourself.